app/schemas/federation_entities.json

```diff
@@ -0,0 +1,366 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "https://diasporafoundation.org/schemas/federation_entities-1.0.0.json",
```

Comment:
> Do you want me to add any documentation on our new JSON features within this PR?

I wont block this PR for the documentation, but I'm still waiting for feedback from @denschub, so if you have the time to add it, do it in the PR. If everything else is finished in the PR, I'll merge it (so I can start with my refactorings), and you can add the documentation later.

> let's make a separate gem.

👍